app/views/groups/members/_create_modal.html.erb

```diff
@@ -9,7 +9,7 @@
         ) %>
       </p>
     </div>
-     <%= form_with(model: @new_member, url: group_members_path, method: :post) do |form| %>
+     <%= form_with(model: @new_member, url: group_members_path(**request.query_parameters), method: :post) do |form| %>
```

Comment:
**question (non-blocking):** Are you adding in the `**request.query_parameters` so that you stay on the same page navigated to in the table when creating a member?

Or is it to just stay on the same tab, i.e. if not on members tab do not load the partial to update the table. If that is the case, I think it is better to use slice to only pull out the `tab` instead of any other parameters.

Reply:
Yup that's exactly what it was being passed in for

Reply:
Updated to instead set the tab in the controller in 9d14e974555ff0bbe5c4b392f2e0b47b8a2b1865